docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -138,6 +138,10 @@ For more information about how the Okta integration manages Teleport resources,
 see the Okta [Architecture](./architecture/) and [User Synchronization](./usersync/)
 guides.
 
+### Configure Access List Synchronization
+
+TODO(mdwn)
```

Comment:
I'm having some difficulty setting things up in my staging tenant, so I need to get images for this so I can fill this out.